### PR TITLE
LPS-69899 Remove remaining deprecated taglibs

### PR DESCRIPTION
**Pull request by Preston-Crary**
Hi @Ithildir,

For some reason the com.liferay.taglib.ui.DiscussionTag.class isn't being copied into the util-taglib-compat's class folder.  All the other ones are working and I checked the 2.4.1 com.liferay.util.taglib.jar from repository.liferay.com and the class is in there so it's unclear to me why only this one isn't being copied.

I'm not really sure how to debug this part of the build more than that, can you look into it and send this to Brian if you get it working?

Thanks again!

cc @shuyangzhou

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:stop

File: modules/core/util-taglib-compat/imported-classes.properties [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+	com.liferay.taglib.ui.AppViewNavigationEntryTag,\
+	com.liferay.taglib.ui.AppViewToolbarTag,\
+	com.liferay.taglib.ui.AssetCategoriesSelectorTag,\
+	com.liferay.taglib.ui.AssetTagsNavigationTag,\
+	com.liferay.taglib.ui.CalendarTag,\
+	com.liferay.taglib.ui.CaptchaTag.java,\
+	com.liferay.taglib.ui.CustomAttributeListTag,\
+	com.liferay.taglib.ui.CustomAttributesAvailableTag,\
+	com.liferay.taglib.ui.CustomAttributeTag,\
+	com.liferay.taglib.ui.DiscussionTag.java,\
```
Comment (Ithildir) [inline modules/core/util-taglib-compat/imported-classes.properties]: The problem is in this `.java` extension. Please remove it and try again 😊

File: modules/core/util-taglib-compat/imported-classes.properties [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+	com.liferay.taglib.portletext.IconBackTag,\
+	com.liferay.taglib.security.EncryptTag,\
+	com.liferay.taglib.theme.LayoutIconTag,\
+	com.liferay.taglib.ui.AppViewDisplayStyleTag,\
+	com.liferay.taglib.ui.AppViewNavigationEntryTag,\
+	com.liferay.taglib.ui.AppViewToolbarTag,\
+	com.liferay.taglib.ui.AssetCategoriesSelectorTag,\
+	com.liferay.taglib.ui.AssetTagsNavigationTag,\
+	com.liferay.taglib.ui.CalendarTag,\
+	com.liferay.taglib.ui.CaptchaTag.java,\
```
Comment (Ithildir) [inline modules/core/util-taglib-compat/imported-classes.properties]: Same here, no `.java` extension!

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/46328.

**Comment by Preston-Crary** [comment]
Ahh, that makes sense.  Thanks!

**Pull request closed without merging**